lms/djangoapps/teams/csv.py

```diff
@@ -321,6 +321,44 @@ def validate_proposed_team_size_wont_exceed_maximum(self, team_name, teamset_id)
         self.user_count_by_team[(teamset_id, team_name)] += 1
         return True
 
+    def remove_user_from_existing_memberships(self, row):
```

Comment:
I think this could use a different name. I really cant think of anything, but 'remove_user_from_existing_memberships' kinda makes it sound like you pass it a user and it removes that user from their memberships.

Idk.. Also, add a note to the docstring about it modifying the input to remove  users that aren't changing.